services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +44,42 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
```

Comment:
Check that the event is not null

Reply:
Also check that the queue handle is not null, return invalid state error if it is